### PR TITLE
adding tippy.js icon

### PR DESCRIPTION
**Pull request by jorgeamadosoria**
tippydotjs

**Issue:**
Adding Tippy.js icon

### Description
Taken from the svg in the home page at https://atomiks.github.io/tippyjs/static/logo-ebc385458e03fdb24af078536af88065.svg
Selected the color from the bubble color, since it is a multicolored icon and no other color could be favored. No guidelines for the icon were provided.
Justified by the project having +5k stars on github: https://github.com/atomiks/tippyjs
Needed for my personal set of used technology icons

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9239,6 +9239,11 @@
             "hex": "ED1C24",
             "source": "https://tinyletter.com/site/press"
         },
+        {
+            "title": "Tippy.js",
+            "hex": "213B50",
+            "source": "https://atomiks.github.io/tippyjs/static/logo-ebc385458e03fdb24af078536af88065.svg"
```
Comment (service-paradis) [inline _data/simple-icons.json:9245]: Did not have time to review the icon yet, but I found the source in their Github repo. Can you use this link instead as source:
https://github.com/atomiks/tippyjs/blob/8f7b2df156ae39c4df23c481b77ab59fe7f08c44/website/src/images/logo.svg

**Review by sachinraja — DISMISSED** [review]
image
Still noticing the differences, not sure if this is within what's acceptable though.

**Review by adamrusted — APPROVED** [review]
This seems to line up perfectly for me. If there are no objections from @simple-icons/icon-review team, we can get this one merged in 😃 

image

**Review by service-paradis — APPROVED** [review]
> This seems to line up perfectly for me. If there are no objections from https://github.com/orgs/simple-icons/teams/icon-review team, we can get this one merged in 😃

If it is ok for you, it is ok for me, feel free to merge this one! 😄

**Review by adamrusted — APPROVED** [review]
Boom, let's get this merged!

**Pull request closed without merging**